hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/JetSqlSerializerHook.java

```diff
@@ -65,7 +63,6 @@ public DataSerializableFactory createFactory() {
         constructors[JSON_VALUE] = arg -> new JsonValueFunction<>();
         constructors[JSON_OBJECT] = arg -> new JsonObjectFunction();
         constructors[JSON_ARRAY] = arg -> new JsonArrayFunction();
-        constructors[JET_SQL_ROW] = arg -> new JetSqlRow();
```

Comment:
Should we guarantee 5.0 and 5.1 members working together for SQL ? If yes, this looks like a non backward compatible change.

Reply:
They work together in a special way: one query is always executed only by a subset of members with equal cluster version. Therefore, `JetSqlRow` instances will always be sent only to members of the same version.